lib/themes.dart

```diff
@@ -538,3 +752,373 @@ class Style extends ThemeExtension<Style> {
+    this.onSecondaryOpacity30,
+    this.onSecondaryOpacity20,
+    required this.background,
+    required this.backgroundAuxiliary,
+    required this.backgroundAuxiliaryLight,
+    required this.backgroundAuxiliaryLighter,
+    required this.backgroundAuxiliaryLightest,
+    required this.onBackground,
+    this.onBackgroundOpacity2,
+    this.onBackgroundOpacity7,
+    this.onBackgroundOpacity13,
+    this.onBackgroundOpacity20,
+    this.onBackgroundOpacity27,
+    this.onBackgroundOpacity33,
+    this.onBackgroundOpacity40,
+    this.onBackgroundOpacity50,
+    this.onBackgroundOpacity56,
+    this.onBackgroundOpacity63,
+    required this.transparent,
+    required this.acceptColor,
+    required this.acceptAuxiliaryColor,
+    required this.declineColor,
+    required this.dangerColor,
+    required this.warningColor,
+    required this.userColors,
+  });
+
+  /// Main [Color] of the application, used to highlight the active interface elements.
+  final Color primary;
+
+  /// Background [Color] for elements associated with the [primary] color.
+  /// For example, the background for buttons, pop-ups, dialog boxes, etc.
+  final Color secondaryBackground;
+
+  /// Light shade of the primary background [Color].
+  final Color secondaryBackgroundLight;
+
+  /// Lightest shade of the primary background [Color].
+  final Color secondaryBackgroundLightest;
+
+  /// [Color] for highlighting UI elements, used to highlight the active elements.
+  final Color secondaryHighlight;
+
+  /// Dark shade of the accent [Color]. Used to create contrast and depth effect.
+  final Color secondaryHighlightDark;
+
+  /// Darkest shade of the main accent [Color]. It is used to emphasize buttons, labels,
+  /// or other user interface elements that should be highlighted and easily visible to the user.
+  final Color secondaryHighlightDarkest;
+
+  /// 20% opacity of the [primary] color.
+  /// Used for - [handRaisedIcon], [element], etc.
+  final Color? secondaryOpacity87;
+
+  /// 15% opacity of the [primary] color.
+  /// Used for - [HintWidget] elements.
+  final Color? secondaryOpacity73;
+
+  ///  [Color] that is used for elements that are displayed on top
+  /// of the main color of the application, for example, text on buttons and icons.
+  final Color onPrimary;
+
+  /// 90% opacity of the [onPrimary] color.
+  /// Used for - [_secondaryTarget] boxes, [DecorationTween], etc.
+  final Color? onPrimaryOpacity7;
+
+  /// 75% opacity of the [onPrimary] color.
+  /// Used for - [BoxDecoration].
+  final Color? onPrimaryOpacity25;
+
+  /// 60% opacity of the [onPrimary] color.
+  /// Used for - [MessageFieldView] containers.
+  final Color? onPrimaryOpacity40;
+
+  /// 50% opacity of the [onPrimary] color.
+  /// Used for - [AcceptVideoButton], [ChewieProgressColors] etc.
+  final Color? onPrimaryOpacity50;
+
+  /// 40% opacity of the [onPrimary] color.
+  /// Used for - [MobileCall] boxes, etc.
+  final Color? onPrimaryOpacity60;
```

Comment:
https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph